OpenRA.Mods.RA2/Traits/CarrierParent.cs

```diff
@@ -100,6 +97,16 @@ public override void InitializeChildEntry(Actor child, BaseSpawnerChildEntry ent
 			carrierChildEntry.SpawnerChild = child.Trait<CarrierChild>();
 		}
 
+		public int GetChildrenInsideCount()
+		{
+			var count = 0;
+			for (var i = 0; i < childEntries.Length; i++)
```

Comment:
Did you have a specific reason for changing this to `for` from the original `foreach`, or maybe using a LINQ `.Count()`?